scripts/compare_benchmarks.py

```diff
@@ -347,9 +348,9 @@ def double_vertical_separators(lines, vertical_separators_to_duplicate):
         + "```diff\n"
     )
     for line in table_string.splitlines():
-        if (green_control_sequence + "+" in line) or ("| Sum " in line and green_control_sequence in line):
+        if (green_control_sequence in line) or ("| Sum " in line and green_control_sequence in line):
```

Comment:
the first one always covers the second one. Not sure what the previous code checked though.

Reply:
Right, that could be simplified now but I also still don't know why we had the `+`-check.

Reply:
@mweisgut @julianmenzler any ideas?

Reply:
After an exciting journey through the git history, I found the following comment: https://github.com/hyrise/hyrise/pull/1819/files#r339463021

At that time, the p-value was still colored green in some cases. This could happen even if the performance did not change. So without the `+`/`-` check, lines could fulfill the condition where only the p-value was green. Since we no longer color p-value green, we can remove the `+`/`-` check here.

Reply:
Great! Thanks a lot for embarking on that journey.

Reply:
Nice digging.